react-app/src/redux/game/actions.js

```diff
@@ -0,0 +1,17 @@
+const actions = {
+  HANDLE_CLICK: 'HANDLE_CLICK',
+  JUMP_TO: 'JUMP_TO'
+};
+
+const actionCreators = {
+  handleClick: squarePosition => ({
+    type: actions.HANDLE_CLICK,
```

Comment:
How about a better name? Like play turn or something like that. Keep in mind in the action you will have tons of actions, an `handleClick` is pretty generic